Update torch-xpu-ops pin (ATen XPU implementation)

fengyuan14 opened the pull request:
Release cycle for PyTorch 2.5
1. Bugfixing: correct reduction logic in cdist kernel.

cc @frank-wei @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @gujinghui @EikanWang @guangyey

EikanWang commented:
@pytorchbot merge -i "Irrelevant to this PR. https://github.com/pytorch/pytorch/issues/135322"

EikanWang commented:
@pytorchbot --help

EikanWang commented:
@pytorchbot merge -i -m "Irrelevant to this PR. https://github.com/pytorch/pytorch/issues/135322"

EikanWang commented:
@pytorchbot merge -f "Irrelevant to this PR. https://github.com/pytorch/pytorch/issues/135322"

fengyuan14 committed:
Update torch-xpu-ops pin (ATen XPU implementation)

Releace cycle for PyTorch 2.5
1. Bugfixing: correct reduction logic in cdist kernel.

Signed-off-by: Feng Yuan <feng1.yuan@intel.com>
